_posts/blog/2017-04-28-doing-things-differently.md

```diff
@@ -0,0 +1,42 @@
+## Doing Things Differently
```

Comment:
Since this is the title, it should go in the front matter as the `title` variable instead of here in the body of the post.